packages/kit/src/exports/vite/utils.js

```diff
@@ -61,16 +44,6 @@ export function deep_merge(...objects) {
 	return result;
 }
 
-/**
- * normalize kit.vite.resolve.alias as an array
- * @param {import('vite').AliasOptions} o
- * @returns {import('vite').Alias[]}
- */
-function normalize_alias(o) {
-	if (Array.isArray(o)) return o;
-	return Object.entries(o).map(([find, replacement]) => ({ find, replacement }));
```

Comment:
anyone knows why we did this previously?

Reply:
vite alias config has a variable type, can be an array or a map https://vitejs.dev/config/shared-options.html#resolve-alias

mergeConfig export of vite takes care of this internally